climada/entity/exposures/base.py

```diff
@@ -118,12 +118,20 @@ class Exposures():
     @property
     def crs(self):
         """Coordinate Reference System, refers to the crs attribute of the inherent GeoDataFrame"""
-        try:
+        if hasattr(self.gdf, "crs") and self.gdf.crs:
             return self.gdf.crs
-        except AttributeError:
-            return self.meta.get('crs')
-
-    def __init__(self, *args, **kwargs):
+        # it is very well possible that gdf.crs is None while the geometry columns has a crs
+        if "geometry" in self.gdf and hasattr(self.gdf.geometry, "crs") and self.gdf.geometry.crs:
+            return self.gdf.geometry.crs
+        return self.meta.get('crs')
+
+    def __init__(self, *args,
+                 meta=None,
+                 tag=None,
+                 ref_year=DEF_REF_YEAR,
+                 value_unit=DEF_VALUE_UNIT,
+                 crs=None,
+                 **kwargs):
```

Comment:
It's a bit more common in CLIMADA to fill the lines with function arguments. If you prefer that, here is how that would look like:
```suggestion
    def __init__(self, *args, meta=None, tag=None, ref_year=DEF_REF_YEAR,
                 value_unit=DEF_VALUE_UNIT, crs=None, **kwargs):
```

Reply:
👍